layer/projects/project_runner.py

```diff
@@ -5,11 +5,12 @@
 from typing import Any, Callable, List, Optional, Sequence
 
 import polling  # type: ignore
+import requests  # type: ignore
```

Comment:
`requests` is a transitive dependency, it should be added to `pyproject.toml`, if used directly. Can we do the upload with `aiohttp`, which sdk already [depends on](https://github.com/layerai/sdk/blob/a32b2903a070561f7e17775a6d2c737d3695d4c7/pyproject.toml#L55)?

Reply:
@donatasm good point -- we also use it in layer/logged_data/log_data_runner.py . I should perhaps change it too (or add requests as a dep)

Reply:
I made the change, but not 100% sure if I did it correctly tbh (Python async stuff is not my forte)